client/webserver/site/src/js/settings.js

```diff
@@ -61,6 +63,27 @@ export default class SettingsPage extends BasePage {
     })
   }
 
+  async showDexInfo (page) {
+    const dexBox = page.dexBox
+    const dexInfo = page.dexInfo.cloneNode(true)
+    const dexInfoAcct = page.dexInfo.querySelector('.dexInfoAcct').cloneNode(true)
+
```

Comment:
If you have multiple dexes, you will be creating multiple elements with the same `id`, which is not allowed. Check out e.g. how I'm using templates at https://github.com/decred/dcrdex/blob/cd855bef19c85f010fbc0f5385ca483b4eeb6490/client/webserver/site/src/js/markets.js#L97